skimage/transform/_geometric.py

```diff
@@ -224,8 +244,81 @@ def residuals(self, src, dst):
         """
         return np.sqrt(np.sum((self(src) - dst) ** 2, axis=1))
 
+    @classmethod
+    def identity(cls, dimensionality=None):
+        """Identity transform
+
+        Parameters
+        ----------
+        dimensionality : {None, 2}, optional
+            This transform only allows dimensionality of 2, where None
+            corresponds to 2. The parameter exists for compatibility with other
+            transforms.
+
+        Returns
+        -------
+        tform : transform
+            Transform such that ``np.all(tform(pts) == pts)``.
+        """
+        return cls(dimensionality=dimensionality)
+
+
+class _HMatrixTransform(_GeometricTransform):
```

Comment:
Not in love with adding another layer of indirection to the inheritance scheme here, but I think it makes sense and I don't see a way around it?

Reply:
It also seems coherent to me - to have a subclass for Transforms based on a homogeneous transform matrix.